Adyen/UI/Form/Items/Basic Items/FormAttributedLabelItem.swift

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import UIKit
+
+/// Simple form item that represent a single attributed UILabel element.
+@_spi(AdyenInternal)
+public class FormAttributedLabelItem: FormItem {
+
+    public var subitems: [FormItem] = []
+
+    public init(originalText: String,
+                link: String,
+                style: TextStyle,
+                linkTextStyle: TextStyle,
+                identifier: String? = nil) {
+        self.identifier = identifier
+        self.style = style
+        self.linkTextStyle = linkTextStyle
+        self.originalText = originalText
+        self.link = link
+    }
+
+    public var identifier: String?
+
+    /// The style of the label.
+    public var style: TextStyle
+
+    /// The style of the link text.
+    public var linkTextStyle: TextStyle
+
+    /// The text of the label.
+    public var originalText: String
+
+    // The link label.
+    public var link: String
+
+    public func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
```

Comment:
Minor: 
This could be useful in future!
Would be nice inject this logic into standalone "Utility" struct and cover with few more tests.

Reply:
As discussed, we can add tests for different strings in new PR.